src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -526,6 +526,19 @@ manila_powermax_server_container=CHANGEME
 manila_powermax_share_data_pools=CHANGEME
 manila_powermax_ethernet_ports=CHANGEME
 
+#
+# PowerFlex cinder parameters
+#
+powerflex_presentation_server_rpm=EMC-ScaleIO-mgmt-server-3.5-0.365.noarch.rpm
+powerflex_gateway_rpm=EMC-ScaleIO-gateway-3.5-0.468.x86_64.rpm
```

Comment:
I wonder if we should move it to PowerFlex settings as they are not cinder config related.
powerflex_presentation_server_rpm=EMC-ScaleIO-mgmt-server-3.5-0.365.noarch.rpm
powerflex_gateway_rpm=EMC-ScaleIO-gateway-3.5-0.468.x86_64.rpm

Reply:
Let's do it differently because it has nothing to with Ansible neither. I'd like to create a section called PowerFlex settings. In this section we'll find three sub sections: 1. DEFAULT for parameters with no specific connection to Cinder nor Ansible 2. Cinder where all parameters related to Cinder will reside and 3. Ansible where all ansible options will appear.

Reply:
+1 I like it

Reply:
It will be part of a next PR.